config/config.go

```diff
@@ -153,13 +150,47 @@ func (cfg *Config) validateServicePipelines() error {
 
 // Service defines the configurable components of the service.
 type Service struct {
+	Telemetry ServiceTelemetry
+
 	// Extensions are the ordered list of extensions configured for the service.
 	Extensions []ComponentID
 
 	// Pipelines are the set of data pipelines configured for the service.
 	Pipelines Pipelines
 }
 
+// ServiceTelemetry defines the configurable settings for service telemetry.
+type ServiceTelemetry struct {
+	Logs ServiceTelemetryLogs
+}
+
+func (srvT *ServiceTelemetry) validate() error {
+	return srvT.Logs.validate()
+}
+
+// ServiceTelemetryLogs defines the configurable settings for service telemetry logs.
+// This MUST be compatible with zap.Config. Cannot use directly zap.Config because
+// the collector uses mapstructure and not yaml tags.
+type ServiceTelemetryLogs struct {
+	// Level is the minimum enabled logging level.
+	Level zapcore.Level
+
+	// Development puts the logger in development mode, which changes the
+	// behavior of DPanicLevel and takes stacktraces more liberally.
+	Development bool
```

Comment:
Does this also enable logging at "Debug" level? If not then this is not equivalent to existing `--log-profile` command line option which may make the transition more difficult.

Reply:
Does it need to be handled with a single config option?  If a combination of two options that can be set together replace a flag that seems fine to me.

Reply:
indeed is a combination of two to replace the flag, but when handle the deprecated flag we will do the right thing.